components/Shop/shop.tsx

```diff
@@ -0,0 +1,285 @@
+    image: Youtube,
+    alt: 'Youtube logo icon',
+  },
+  {
+    href: 'https://gmail.com',
+    image: Email,
+    alt: 'Email logo icon',
+  },
+]
+
+export default function Shop() {
+  return (
+    <View
+      as="section"
+      css={{
+        backgroundColor: '$white1',
+      }}
+    >
+      <Container
+        size="large2"
+        css={{
+          gridTemplateRows: 'auto 1fr',
+          gridTemplateAreas: ' "carousel" "content"',
+          padding: 0,
+          height: '82vh',
+          '@tablet': {
+            height: '78vh',
+            gridTemplateColumns: '50vw 1fr',
+            gridTemplateAreas: '"carousel content content"',
+          },
+        }}
+        as={Grid}
+      >
+        <ImageContainer>
+          <Image src={Logo} alt="logo" objectFit="contain" />
+        </ImageContainer>
+        <ShopContent />
+      </Container>
+      <Wave />
+    </View>
+  )
+}
+
+function ShopContent() {
+  const [email, setEmail] = useState('')
+  const router = useRouter()
+
+  const handleSubmit = async (event: SyntheticEvent) => {
+    event.preventDefault()
+    await subscribeToBlog(email)
+    setEmail('')
+    router.push('/thank-you')
+  }
+  return (
+    <Flex
+      direction="column"
+      css={{
+        gridArea: 'content',
+        px: '$5',
+        pt: '$4',
+        '@desktop': { pt: '10vh' },
+      }}
+      gap="3"
+    >
+      <HeaderText
+        size={{
+          '@initial': '6',
+          '@tablet': '8',
+          '@desktop': '9',
+        }}
+      >
+        Coming Soon
+      </HeaderText>
+      <SloganText
+        size={{
+          '@initial': '7',
+          '@tablet': '9',
+          '@desktop': '12',
+        }}
+      >
+        Get Notified when we launch!
+      </SloganText>
+      <Flex
+        as="form"
+        direction="column"
+        align={{ '@initial': 'center', '@tablet': 'start' }}
+        onSubmit={handleSubmit}
+      >
+        <InputWrapper as={Flex} align="center">
+          <BaseInput
+            required
+            type="email"
+            variant="unstyled"
+            placeholder="@E-mail"
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
+          />
+          <Button variant="primary" fitContent>
+            <ButtonText>Notify</ButtonText>
+            <ButtonText>Me</ButtonText>
+            <ArrowRightIcon width={16} fill="white" />
+          </Button>
+        </InputWrapper>
+      </Flex>
+
+      <Subtitle
+        size={{
+          '@initial': '2',
+          '@tablet': '3',
+          '@desktop': '5',
+        }}
+      >
+        *For the meantime, use our application!
+      </Subtitle>
+      <FlexRow gap="3" justify={{ '@initial': 'center', '@tablet': 'start' }}>
+        <Link href="http://facebok.com" target="_blank">
+          <Download>
+            <Image src={Appstore} alt="appstore" />
+          </Download>
+        </Link>
+        <Link href="http://facebok.com" target="_blank">
```

Comment:
please ko link ani sa felicity na ayaaa here ang link
https://play.google.com/store/apps/details?id=com.felicityincorporated.felicity

Reply:
also please ko add ani na prop ayaaa for security ni `rel="noreferrer noopener"`
```suggestion
        <Link href="http://facebok.com" target="_blank" rel="noreferrer noopener">
```